packages/asset-swapper/package.json

```diff
@@ -66,7 +66,7 @@
         "@0x/contracts-zero-ex": "^0.31.0",
         "@0x/dev-utils": "^4.2.11",
         "@0x/json-schemas": "^6.4.1",
-        "@0x/neon-router": "^0.3.2",
+        "@0x/neon-router": "0xProject/gitpkg-registry#0x-neon-router-v0.3.2-commit.4b8eda4-4b8eda4",
```

Comment:
Will update this with the proper version once the code has been reviewed, and I have made a proper publish